### ISSUE TITLE
Deprecate package for eslint-plugin-vue-a11y

### ISSUE THREAD

**Issue by goatandsheep** [maranran/eslint-plugin-vue-accessibility#1]
This package seems less updated and starred than [`eslint-plugin-vue-a11y`](https://github.com/maranran/eslint-plugin-vue-a11y). I think this package should be deprecated in favour of it
